app/test_main.py

```diff
@@ -1 +1,68 @@
-# write your code here
+from unittest import mock
+import pytest
+
+
+from app.main import can_access_google_page
+
+
+@pytest.mark.parametrize(
```

Comment:
You should parametrize the function that you are going to test and pass this data to.